src/Controllers/Library/search.php

```diff
@@ -1,6 +1,54 @@
 <?php
-
+/**
+ * Allows URY Librarians to search for Tracks.
+ */
 use \MyRadio\MyRadio\CoreUtils;
+use \MyRadio\MyRadio\URLUtils;
+use \MyRadio\ServiceAPI\MyRadio_Track;
+use \MyRadio\MyRadio\MyRadioForm;
+use \MyRadio\MyRadio\MyRadioFormField;
+
+$form = new MyRadioForm(
+    'lib_search',
+    'Library',
+    'search',
```

Comment:
They are set by the root initialise on the request. The reason for them is to tell the form where to submit. It probably makes most sense to leave them as is on second thought